frontend/src/utils/image-utils.ts

```diff
@@ -0,0 +1,190 @@
+    case 'right':
+      // Right Diagonal lines svg (comes to left)
+      return `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" width="50" height="50"><defs><style>.cls-1,.cls-3{fill:none;}.cls-2{clip-path:url(#clip-path);}.cls-3{stroke:${strokeColor};stroke-miterlimit:10;stroke-width:2px;}.cls-4{fill:url(#_19-2_black_diagonal);}</style><clipPath id="clip-path"><rect id="SVGID" class="cls-1" width="100" height="100"/></clipPath><pattern id="_19-2_black_diagonal" data-name="19-2 black diagonal" width="100" height="100" patternTransform="translate(-13.41 -30.89)" patternUnits="userSpaceOnUse" viewBox="0 0 100 100"><rect class="cls-1" width="100" height="100"/><g class="cls-2"><line class="cls-3" x1="-13.41" y1="53.03" x2="93.66" y2="-53.03"/><line class="cls-3" x1="-7.86" y1="58.59" x2="99.22" y2="-47.48"/><line class="cls-3" x1="-2.3" y1="64.14" x2="104.78" y2="-41.92"/><line class="cls-3" x1="3.26" y1="69.7" x2="110.34" y2="-36.37"/><line class="cls-3" x1="8.81" y1="75.26" x2="115.89" y2="-30.81"/><line class="cls-3" x1="14.37" y1="80.81" x2="121.44" y2="-25.26"/><line class="cls-3" x1="19.92" y1="86.37" x2="126.99" y2="-19.7"/><line class="cls-3" x1="25.48" y1="91.92" x2="132.55" y2="-14.14"/><line class="cls-3" x1="31.03" y1="97.48" x2="138.1" y2="-8.59"/><line class="cls-3" x1="36.59" y1="103.03" x2="143.66" y2="-3.03"/><line class="cls-3" x1="42.14" y1="108.59" x2="149.21" y2="2.52"/><line class="cls-3" x1="47.7" y1="114.14" x2="154.77" y2="8.08"/><line class="cls-3" x1="53.26" y1="119.7" x2="160.33" y2="13.63"/><line class="cls-3" x1="58.81" y1="125.26" x2="165.88" y2="19.19"/><line class="cls-3" x1="64.37" y1="130.81" x2="171.44" y2="24.74"/><line class="cls-3" x1="69.92" y1="136.37" x2="176.99" y2="30.3"/><line class="cls-3" x1="75.48" y1="141.92" x2="182.55" y2="35.86"/><line class="cls-3" x1="81.03" y1="147.48" x2="188.1" y2="41.41"/><line class="cls-3" x1="86.59" y1="153.03" x2="193.66" y2="46.97"/></g></pattern></defs><title>Asset 8</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><rect class="cls-4" width="100" height="100"/></g></g></svg>`;
+    default:
```

Comment:
You don't need to duplicate the left shape. simply have case "right" and default to left.